pipelines/measurement/measurement_tract.yaml

```diff
@@ -20,3 +23,6 @@ tasks:
         config.measure.retarget(TExTask)
         config.measure.minSep = 5.0
         config.measure.maxSep = 20.0
+        config.measure.column = "slot_Shape"
```

Comment:
It looks like the values being set here for `column` and `columnPsf` are the same as the defaults, so suggest that we don't specify them again in the yaml file.